doc/whats_new.rst

```diff
@@ -209,6 +209,8 @@ Enhancements
 
 Bug fixes
 .........
+   - Fixed a bug where :class:`_PLS` returns a 2D-array when a 1D-array is expected
```

Comment:
It seems CCA and PLSRegression are the only two public classes that inherit _PLS.

This PR does not the CCA API, only PLSRegression.

Maybe I should write 

`- Fixed a bug where :class:`PLSRegression` returns a 2D-array when a 1D-array is expected`